apim/3.x/values.yaml

```diff
@@ -585,6 +586,7 @@ api:
         kubernetes.io/ingress.class: nginx
         nginx.ingress.kubernetes.io/configuration-snippet: "etag on;\nproxy_pass_header ETag;\nproxy_pass_header if-match;\n"
         # kubernetes.io/tls-acme: "true"
+      #ingressClassName: nginx
```

Comment:
As `nginx` is already the default value for the `kubernetes.io/ingress.class` annotation, I think it's probably better to uncomment this line (and the others). 

wdyt ?

Reply:
yeah. lets comment the `kubernetes.io/ingress.class: nginx` annotation. 
== `ingressClassName: nginx` should be commented in instead?

Reply:
As it is already mentioned in the ticket, you can NOT have both the annotation and ingressClassName together but from what I see you just added the ingressClassName and without doing something about the "kubernetes.io/ingress.class: nginx" annotation or maybe I missed something?

Reply:
IngressClass is THE way to define Ingress objects to the Ingress Controller on currently supported kubernetes versions.

Indeed annotations for ingress class has to be commented to allow old k8s versions to be supported and to be compliant with the end of deprecation (=removal) for ingress class annotation in 1.24+

No need to make a switch depending on the version because it would confuse parameters settings.